nhais/outbound/request/acceptance_amendment.py

```diff
@@ -0,0 +1,105 @@
+from json import JSONDecodeError
+from typing import Any
+
+import tornado.web
+from fhir.resources.codeableconcept import CodeableConcept
+from fhir.resources.coding import Coding
+from fhir.resources.fhirabstractbase import FHIRValidationError
+from fhir.resources.operationoutcome import OperationOutcome
+from fhir.resources.operationoutcome import OperationOutcomeIssue
+
+from tornado import httputil
+from outbound.schema.schema_validation_exception import SchemaValidationException
+
+from utilities import integration_adaptors_logger as log, timing
+from utilities import message_utilities
+
+from outbound.schema import validate_request
+import json
```

Comment:
Run an "Optimize Imports" on the nahis/ folder. It will reorder these to be PEP 8 style